server/seed.js

```diff
@@ -31,15 +38,12 @@ function createItem(name, price, cb) {
   });
 }
 
-function createOrder(id, status, items, cb) {
-  const order = Order({ id, status });
-
-  order.entries = items.map((item) => ({
-    item_id: mongoose.Types.ObjectId(item.id),
-    comment: faker.lorem.sentence()
-  }));
-
-  order.save(cb);
+function createOrder(items, cb) {
+  Order({
+    id: faker.random.number(),
+    status: _.sample(orderStatuses),
+    entries: items.map(toEntry)
```

Comment:
What do you think about putting toEntry, toOrder, etc. into some sort of parser.js?